packages/client/src/client.ts

```diff
@@ -62,7 +56,7 @@ export class DefaultRoutingV1HttpApiClient implements RoutingV1HttpApiClient {
     this.started = false
   }
 
-  async * getProviders (cid: CID, options: AbortOptions | undefined = {}): AsyncGenerator<PeerInfo, any, unknown> {
+  async * getProviders (cid: CID, options: AbortOptions | undefined = {}): AsyncGenerator<Record, any, unknown> {
```

Comment:
`getProviders` vs `findProviders`: I'm fine with both (in Go it's `FindProviders`)